components/telemetry-operator/apis/telemetry/v1alpha1/tracepipeline_types.go

```diff
@@ -56,8 +56,73 @@ func (b *BasicAuthOptions) IsDefined() bool {
 	return b.User.IsDefined() && b.Password.IsDefined()
 }
 
+type TracePipelineConditionType string
+
+// These are the valid statuses of LogPipeline.
+const (
+	TracePipelinePending TracePipelineConditionType = "Pending"
+	TracePipelineRunning TracePipelineConditionType = "Running"
+)
+
+const (
```

Comment:
In the LogPipeline status we also have a condition for a missing secret that is referenced. We should add this here as well since that might be an important information for the user. You can find the implementation for the LogPipeline [here](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/controller/logpipeline/status.go#L59).

Reply:
secret reference check added